VisualPIC/DataReading/rawDataReaders.py

```diff
@@ -193,7 +193,13 @@ def _ReadTime(self, timeStep):
 
     def _ReadUnits(self):
         # OpenPMD data always provide conversion to SI units
-        self.dataUnits = "a.u." # TODO: Get the correct unit
+        # TODO: Get the units from file
+        if self.internalName == "x" or self.internalName == "y" or self.internalName == "z":
+            self.dataUnits = "μm"
```

Comment:
the unit is not defined at this point of raw data reading. Some codes might be microns, some might not.
In the future we have an optional hint via https://github.com/openPMD/openPMD-standard/issues/149

Reply:
I thought OpenPMD-viewer always returned spatial particle data in microns. If this depends on the code then there should be a way of determining the units from the file metadata, which would be ideal solution anyways, otherwise VisualPIC wont be able to handle correctly the unit conversion.

Reply:
@ax3l : Here all the data is obtained through `openPMD-viewer` (as opposed to reading the file directly with `h5py`). Therefore all the data is already automatically converted to SI units (and in the case of particle positions, to microns). So I think that `self.dataUnits = "μm"` is correct here. Do you agree?